9control.tex

```diff
@@ -1189,8 +1189,8 @@ \subsubsection{Recruitment Deviation Setup}
 
 	1 \Tstrut & Do Recruitment Deviations & This selects the way in which recruitment deviations are coded: \\
 	  & & 0: None (so all recruitments come from spawner recruitment curve). \\
-	  & & 1: Deviation vector (previously the only option): the deviations are encoded as a deviation vector, so ADMB enforces a sum-to-zero constraint. \\
```

Comment:
@Rick-Methot-NOAA and @iantaylor-NOAA  I have added some text here to try to be more explicit with the hope that it answers some of the questions that Paul Starr has been asking. Can you review this section in regards to that?

Reply:
1: Deviation vector:  ADMB enforces a sum-to-zero constraint, but only in the main period.  This is a commonly used option but is unreliable with MCMC.  Option 2 is now recommended.